lucida/commandcenter/app.py

```diff
@@ -43,7 +42,7 @@ def flask_listener():
                 threaded=True)
 
 def web_socket_listener():
-    print 'Start web socket at ' + str(cmd_port)
+    print 'Start web socket at 8081' 
```

Comment:
It would be better to have an interface for Lucida core to manage the port in future. Right now the task is just to make it work.